microsoft-edge/webview2/release-notes.md

```diff
@@ -19,6 +19,69 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > WebView2 bug fixes are either Runtime or SDK specific.  
 
+## 1.0.955-prerelease
+
+Release Date: July 26, 2021  
+
+[NuGet package][NuGetGallery1.0.955-prerelease] \| Minimum Microsoft Edge version to load: 86.0.616.0 or newer \| Full API Compatibility: 93.0.955.0 or newer  
```

Comment:
Can we make a one-time change to Full API Compatibility to 93.0.967.0 for 1.0.955-prerelease? This is for a bug fixing usage of `ICoreWebView2_5::add_ClientCertificatedRequested` that @vickiez just checked into main.

```suggestion
[NuGet package][NuGetGallery1.0.955-prerelease] \| Minimum Microsoft Edge version to load: 86.0.616.0 or newer \| Full API Compatibility: 93.0.967.0 or newer  
```